database/migrations/Version20210322125634.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Database\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+
+class Version20210322125634 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE regions (uid INT AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, polygon geometry NOT NULL, PRIMARY KEY(uid)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE users_regions (idusers_regions INT AUTO_INCREMENT NOT NULL, user INT(11) NOT NULL, region INT(11) NOT NULL, PRIMARY KEY(idusers_regions), UNIQUE INDEX users_region_unique_idx(user, region)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
```

Comment:
Noting here that the id column is named idusers_regions - any particular reason?

Perhaps in retro we can have a brief chinwag about what we think DB naming conventions should be.

Reply:
I went for consistency with `idusers_groups` in the restarters `users_groups` table.

Personally I would use `id` here too, but...